translate/cloud-client/src/main/java/com/example/translate/HybridGlossaries.java

```diff
@@ -0,0 +1,301 @@
+    }
+  }
+  // [END translate_hybrid_vision]
+
+  // [START translate_hybrid_create_glossary]
+  /** Creates a GCP glossary resource
+   * Assumes you've already manually uploaded a glossary to Cloud Storage
+   *
+   * ARGS
+   * @param languages list of languages in the glossary
+   * @param projectId GCP project id
+   * @param glossaryDisplayName name you want to give this glossary resource
+   * @param glossaryUri the uri of the glossary you uploaded to Cloud Storage
+   * 
+   **/
+  static void createGlossary(List<String> languages, String projectId,
+                             String glossaryDisplayName, String glossaryUri) {
+    try (TranslationServiceClient translationServiceClient = TranslationServiceClient.create()) {
+
+      LocationName locationName =
+              LocationName.newBuilder().setProject(projectId).setLocation("us-central1").build();
+
+
+      LanguageCodesSet languageCodesSet =
+              LanguageCodesSet.newBuilder().addAllLanguageCodes(languages).build();
+
+      GcsSource gcsSource = GcsSource.newBuilder().setInputUri(glossaryUri).build();
+
+      GlossaryInputConfig glossaryInputConfig =
+              GlossaryInputConfig.newBuilder().setGcsSource(gcsSource).build();
+
+      GlossaryName glossaryName =
+              GlossaryName.newBuilder()
+                      .setProject(projectId)
+                      .setLocation("us-central1")
+                      .setGlossary(glossaryDisplayName)
+                      .build();
+
+      Glossary glossary =
+              Glossary.newBuilder()
+                      .setLanguageCodesSet(languageCodesSet)
+                      .setInputConfig(glossaryInputConfig)
+                      .setName(glossaryName.toString())
+                      .build();
+
+      CreateGlossaryRequest request =
+              CreateGlossaryRequest.newBuilder()
+                      .setParent(locationName.toString())
+                      .setGlossary(glossary)
+                      .build();
+
+      // Call the API
+      Glossary response = translationServiceClient.createGlossaryAsync(request)
+              .get(300, TimeUnit.SECONDS);
+
+      System.out.format("Created: %s\n", response.getName());
+    } catch (Exception e) {
+      System.out.format("No new glossary was created.\n%s\n", e.getMessage());
+    }
+  }
+  // [END translate_hybrid_create_glossary]
+
+  // [START translate_hybrid_translate]
+  /**
+   * Translates text to a given language using a glossary
+   *
+   * ARGS
+   * text: String of text to translate
+   * sourceLanguageCode: language of input text
+   * targetLanguageCode: language of output text
+   * projectId: GCP project id
+   * glossaryName: name you gave your project's glossary
+   *     resource when you created it
+   * RETURNS
+   * String of translated text
+   **/
+  static String translateText(String text, String sourceLanguageCode, String targetLanguageCode,
```

Comment:
This should be a [style violation](https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break).

Consider running [google-java-format](https://github.com/google/google-java-format).